docs/config.md

```diff
@@ -129,6 +129,30 @@ php_server [<matcher>] {
 }
 ```
 
+### Full Duplex (h1)
+
+When using http 1.x, it may be desirable to enable full-duplex mode to allow writing a response before the entire body
+has been read. (for example: websockets, SSE, etc.)
```

Comment:
```suggestion
has been read. (for example: WebSocket, Server-Sent Events, etc.)
```

Nit: is it really necessary for WebSocket?

Reply:
I haven't tested websockets directly, but I assume this is related to the larvel live wire issues we were seeing.

Reply:
I don't think websockets is affected. That's a different code path if using `reverse_proxy`, the connection gets hijacked and turned into a pipe.

Also Livewire doesn't use websockets btw, there's no "push from the server" with it, it's basically just HTML replacement on the clientside in response to ajax requests.